recipes/lda/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "lda" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 50b91f80aff1b138b8b78f48d581c0e685e7adb393da9e45905773d5f4cc8bf5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py>=37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+  host:
+    - pip
+    - python
+    - numpy
+    - pbr
+    - setuptools
+    
+  run:
+    - python
```

I think you also need ``numpy`` here, am I right?
```suggestion
    - python
    - numpy
```